codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/ValueEntitySourceGeneratorSuite.scala

```diff
@@ -16,23 +16,38 @@
 
 package com.akkaserverless.codegen.scalasdk
 
+import com.google.protobuf.compiler.PluginProtos.CodeGeneratorRequest
 import com.lightbend.akkasls.codegen.TestData
+import protocgen.CodeGenRequest
+import scalapb.compiler.{ DescriptorImplicits, GeneratorParams }
 
 class ValueEntitySourceGeneratorSuite extends munit.FunSuite {
   import com.akkaserverless.codegen.scalasdk.impl.ValueEntitySourceGenerator._
+  implicit val descriptorImplicits =
+    DescriptorImplicits.fromCodeGenRequest(
+      GeneratorParams(),
+      CodeGenRequest(
+        parameter = "",
+        filesToGenerate = Seq.empty,
+        allProtos = Seq.empty,
+        compilerVersion = None,
+        CodeGeneratorRequest.newBuilder().build()))
 
   test("it can generate a value entity implementation skeleton") {
     val file = generateImplementationSkeleton(TestData.valueEntity(), TestData.simpleEntityService())
     assertNoDiff(
       file.content,
       s"""package com.example.service.domain
          |
-         |
+         |import com.example.service.GetValue
+         |import com.example.service.MyState
+         |import com.example.service.SetValue
+         |import com.external.Empty
          |
          |class MyValueEntity /* extends AbstractMyValueEntity */ {
-         |  def set(currentState: Unit, command: Unit): Unit = ???
+         |  def set(currentState: Unit, command: SetValue): Empty = ???
          |
-         |  def get(currentState: Unit, command: Unit): Unit = ???
+         |  def get(currentState: Unit, command: GetValue): MyState = ???
```

Comment:
`currentState: Unit`? Is that still a missing piece?

Reply:
Correct! Referring to the input and output messages of a service now works, but there's still something going wrong with the state model, likely because it is a reference from a custom annotation. That's the next I'm going to pick up, but can be a separate PR I think